aten/src/ATen/native/cuda/Dropout.cu

```diff
@@ -321,14 +319,16 @@ inline void launcher(
 
 } //anonymous namespace
 
+template <typename mask_t>
 std::tuple<Tensor,Tensor>
-fused_dropout_cuda(const Tensor& self, double p, c10::optional<Generator> gen_){
-  auto gen = get_generator_or_default<CUDAGeneratorImpl>(gen_, cuda::detail::getDefaultCUDAGenerator());
-  Tensor ret = at::empty_like(self);
-  Tensor mask = at::empty_like(self, self.options().dtype(kByte));
+dropout_cuda(CUDAGeneratorImpl* gen, const Tensor& self, double p, bool train){
```

Comment:
you don't need `train` argument here, non-training case can be handled in `native_dropout_cuda`.

Reply:
Sounds fair, will update that.